solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBSolrClient.java

```diff
@@ -480,35 +488,70 @@ public RequestWriter getRequestWriter() {
     return requestWriter;
   }
 
+  private boolean isServerAlive(ServerWrapper zombieServer)
+      throws SolrServerException, IOException {
+    if (null == aliveCheckQuery) {
```

Comment:
I think we have normally a check to see if we are in debug mode before calling the `.debug`